internal/fulcio/identity.go

```diff
@@ -26,25 +26,64 @@ import (
 	"fmt"
 	"io"
 	"os"
+	"path/filepath"
 
+	"github.com/sigstore/cosign/cmd/cosign/cli/fulcio/fulcioroots"
```

Comment:
```suggestion
	"github.com/sigstore/sigstore/pkg/fulcioroots"
```

This will also mean `fulcioroots.Get()` returns an err.

Reply:
Done!